### PR TITLE
Fix hover state for all plain variant icons

### LINKED ISSUE #4552: Add hover states to plain toggle variants
Should be consistent across all menu like components:
Dropdown, options menu, select, menu toggle, etc...

----

re: https://github.com/patternfly/patternfly-react/pull/6627#issuecomment-982697224

> It will be subtle but consistent with our other icon action patterns. It would have the same colors/states, secondary text color (default) -> primary text color (hover) just to give it a small sense of feedback.

image

### PR DESCRIPTION
**Pull request by srambach**
Fixes #4552 

This changed the toggle icon to be darker on hover for dropdown, options menu, select, menu toggle, context selector, and pagination.

### CODE REVIEW

File: src/patternfly/components/MenuToggle/menu-toggle.scss [hunk 1]
```diff
@@ -98,6 +98,12 @@
 
   // Toggle icon
   --pf-c-menu-toggle__toggle-icon--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-menu-toggle__toggle-icon--Color: var(--pf-c-menu-toggle--Color);
+  --pf-c-menu-toggle--m-plain__toggle-icon--Color: var(--pf-global--Color--200);
+  --pf-c-menu-toggle--m-plain--hover__toggle-icon--Color: var(--pf-global--Color--100);
+  --pf-c-menu-toggle--m-plain--active__toggle-icon--Color: var(--pf-global--Color--100);
+  --pf-c-menu-toggle--m-plain--focus__toggle-icon--Color: var(--pf-global--Color--100);
+  --pf-c-menu-toggle--m-plain--expanded__toggle-icon--Color: var(--pf-global--Color--100);
```
Comment (mcoker) [inline src/patternfly/components/MenuToggle/menu-toggle.scss:106]:
```suggestion
  --pf-c-menu-toggle--m-plain--expanded__toggle-icon--Color: var(--pf-global--Color--100);
```
```suggestion
  --pf-c-menu-toggle--m-plain--m-expanded__toggle-icon--Color: var(--pf-global--Color--100);
```

File: src/patternfly/components/MenuToggle/menu-toggle.scss [hunk 2]
```diff
@@ -197,43 +203,51 @@
...
     --pf-c-menu-toggle--after--BorderBottomColor: var(--pf-c-menu-toggle--active--after--BorderBottomColor);
+    --pf-c-menu-toggle--m-plain__toggle-icon--Color: var(--pf-c-menu-toggle--m-plain--active__toggle-icon--Color);
   }
 
   &.pf-m-expanded {
     --pf-c-menu-toggle--Color: var(--pf-c-menu-toggle--m-expanded--Color);
     --pf-c-menu-toggle--BackgroundColor: var(--pf-c-menu-toggle--m-expanded--BackgroundColor);
     --pf-c-menu-toggle--after--BorderBottomWidth: var(--pf-c-menu-toggle--m-expanded--after--BorderBottomWidth);
     --pf-c-menu-toggle--after--BorderBottomColor: var(--pf-c-menu-toggle--m-expanded--after--BorderBottomColor);
+    --pf-c-menu-toggle--m-plain__toggle-icon--Color: var(--pf-c-menu-toggle--m-plain--expanded__toggle-icon--Color);
```
Comment (mcoker) [inline src/patternfly/components/MenuToggle/menu-toggle.scss:250]:
```suggestion
    --pf-c-menu-toggle--m-plain__toggle-icon--Color: var(--pf-c-menu-toggle--m-plain--m-expanded__toggle-icon--Color);
```

**Review by mcoker — CHANGES_REQUESTED** [review]
Fantastic! Just one var name that needs to be updated.

**Review by mcoker — APPROVED** [review]

**Comment by mattnolting** [comment]
Looks like the primary/secondary dropdown caret changed

Screen Shot 2022-01-19 at 5 07 38 PM

**Comment by mattnolting** [comment]
Plain menu toggle doesn't have a color change

Screen Shot 2022-01-19 at 5 10 11 PM

**Comment by mattnolting** [comment]
Options menu hover doesn't seem to have a change

Screen Shot 2022-01-19 at 5 14 14 PM

**Comment by srambach** [comment]
> Plain menu toggle doesn't have a color change
> 
This is pre-existing and I don't think should be part of this PR. 

I think the primary/secondary dropdown color is fixed now.
I am seeing a color change on the options menu hover, are you sure you don't? @mattnolting 

Thanks for the inherit fallback idea @mcoker that works nicely.

**Review by mcoker — APPROVED** [review]
🥳

opened an issue about the plain menu toggle state colors here - https://github.com/patternfly/patternfly/issues/4630

**Review by mattnolting — APPROVED** [review]
LGTM!

**Pull request merged**